wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <v-container>
+    <div class="pyro">
+      <div class="before"></div>
+      <div class="after"></div>
+    </div>
+
+    <v-progress-linear
+      v-if="isLeaderboardLoading"
+      class="mx-auto"
+      color="primary"
+      height="10"
+      indeterminate
+      rounded
+      width="75%"
+    />
+    <v-card variant="flat" class="pa-3 bg-secondary" v-else>
+      <v-card-title class="text-center text-h5 mb-3">
+        <v-icon class="rotate2d" color="primary">mdi-star</v-icon>
+        Leaderboard
+        <v-icon class="rotate2d" color="primary">mdi-star</v-icon>
+      </v-card-title>
+
+      <v-table>
+        <thead>
+          <tr>
+            <th
+              style="position: sticky; left: 0"
+              class="text-center font-weight-bold bg-primary"
+            >
+              Rank
+            </th>
+            <th
+              style="position: sticky; left: 50px"
+              class="text-center font-weight-bold text-no-wrap bg-primary"
+            >
+              Player
+            </th>
+            <th class="text-center font-weight-bold text-no-wrap bg-primary">
```

Comment:
```suggestion
            <th class="text-center font-weight-bold bg-primary">
```
Letting the words wrap in these headers would allow for better UX on smaller (mobile) screens.

Reply:
good to know!